client/src/lib/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface Recipe {
+  name: string
+  relatedCharacters: Array<string>
+  ingredients: Array<string>
+  description: string
+  image: any
```

Comment:
shouldnt this be string looks like its being used like one

Reply:
Fixed this, thanks!